### PR TITLE
adding template for additional env variables

### PR DESCRIPTION
**Pull request by gkarthiks**
**What this PR does / why we need it**:
Adding the template for the additional environment variables which can be used to accommodate the password to connect to the redis database.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
N/A

**Special notes for your reviewer**:
@acondrat

### CODE REVIEW

File: stable/prometheus-redis-exporter/values.yaml [hunk 1]
```diff
@@ -16,6 +16,13 @@ image:
   tag: v0.16.0
   pullPolicy: IfNotPresent
 extraArgs: {}
+# Additional Environment variables
+env:
+  # - name: REDIS_PASSWORD
```
Comment (davidkarlsen) [inline stable/prometheus-redis-exporter/values.yaml]: Should it be initialized to empty map by default?
Better update Readme too for documentation

Comment (gkarthiks) [reply stable/prometheus-redis-exporter/values.yaml]: @davidkarlsen  updated the README and initialized the *env* with empty map for better understanding. please review and if anything else let me know.

**Pull request closed without merging**